packages/pwa-kit-cli/bin/cli.js

```diff
@@ -189,21 +188,6 @@ const main = () => {
             })
         })
 
-    program
-        .command('lint')
-        .description('lint all source files')
-        .argument('<path>', 'path or glob to lint')
-        .option('--fix', 'Try and fix errors (default: false)')
-        .action((path, {fix}) => {
-            const eslint = p.join(require.resolve('eslint'), '..', '..', '..', '.bin', 'eslint')
-            const eslintConfig = p.join(__dirname, '..', 'configs', 'eslint', 'eslint-config.js')
-            execSync(
-                `${eslint} --config ${eslintConfig} --resolve-plugins-relative-to ${pkgRoot}${
-                    fix ? ' --fix' : ''
-                } "${path}"`
-            )
-        })
-
     program
         .command('format')
```

Comment:
linting and formatting usually go together. Can the `format` command also be removed?

Reply:
So out eslint configurations uses the `prettier` plugin. I would agree that it's probably not required to have an additional command to only run prettier when it can be run via lint:fix. I don't know if the distinction between the two is enough to warrant a separate command.